### ISSUE TITLE
[Bug]: Missing kubelet and kubeproxy plugins on child's dashboards

### ISSUE THREAD

**Issue by Garahk** [netdata/netdata#14394]
### Bug description

Environment consist of a K8s cluster with 3 nodes, kind-control-plane and 2 workers. Ever since updated netdata to **v1.37.1** (agent-sd to v.0.2.7) child's are missing the `kubelet` and `kubeproxy` plugins. Myself particularly use the chart `Bytes Used By The Pod Logs On The Filesystem (k8s_kubelet.kubelet_pods_log_filesystem_used_bytes)`  for custom purposes.
Cluster is mounted on a Kind container.

### Expected behavior

Kubelet and Kubeproxy plugins running

![image](https://user-images.githubusercontent.com/99432183/215894691-8d47d9e0-5430-48ea-a6ad-9f1f0016a6ae.png)

### Steps to reproduce

1. Updated Netdata to v1.37.1
2. Missing Kubelet and Kubeproxy plugins

### Installation method

helmchart (kubernetes)

### System info

```shell
Linux netdata-parent-5ff7874644-2t9rk 3.10.0-1160.76.1.el7.x86_64 #1 SMP Tue Jul 26 14:15:37 UTC 2022 x86_64 Linux
/etc/alpine-release:3.16.3
/etc/os-release:NAME="Alpine Linux"
/etc/os-release:ID=alpine
/etc/os-release:VERSION_ID=3.16.3
/etc/os-release:PRETTY_NAME="Alpine Linux v3.16"
```

### Netdata build info

```shell
bash-5.1# netdata -W buildinfo
Version: netdata v1.37.1
Configure options:  '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libexecdir=/usr/libexec' '--libdir=/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--without-bundled-protobuf' '--disable-dependency-tracking' '--disable-ebpf' 'CFLAGS=-O2 -pipe' 'LDFLAGS='
Install type: oci
    Binary architecture: x86_64
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    NO
    IPMI:                    YES
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 YES
    Prometheus Remote Write: YES
Debug/Developer Features:
    Trace Allocations:       NO
```

**Comment by ilyam8** [comment]
@Garahk thanks for reporting. Fixed in netdata/helmchart#333.

**Comment by Garahk** [comment]
> @Garahk thanks for reporting. Fixed in [netdata/helmchart#333](https://github.com/netdata/helmchart/pull/333).

Awesome, I tested branch ilyam8:fix_child_config and it works perfectly, just as expected 

![image](https://user-images.githubusercontent.com/99432183/216131863-06ca6948-e50e-4a9e-aa09-f0bd7ee74b5a.png)

Just waiting for merge to be approved to close issue.
